### DISCUSSION TITLE
Cue is empty for some playlists

### DISCUSSION THREAD

**Discussion by jdekoven** [punxaphil/custom-sonos-card#537] [Q&A]
Just updated to the new Sonos Card and excited about being able to see the cue and select what to play etc...  I'm using You Tube Music and it seems like some playlists will show the que and some won't.  If I create a new list it will show the que, but if I transfer songs to it from an old list that doesn't show the que, then the new one will stop as well.  So are some of the tracks doing this?  Would be a pain to rebuild from scratch, but just want to see if anyone has experienced this.

**Comment by punxaphil** [comment]
Please go to Developer tools -> Actions and run the sonos.get_queue action and check if it behaves ok there

**Comment by jdekoven** [comment]
Unfortunately it doesn't.  It seems like some of the lists or songs are creating the problem.  I had one list that didn't work so I removed all the songs, then put a few back and it works.  So may have to kind of rebuild them.  If you have any other thoughts let me know.  Thanks for your help!

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by punxaphil** [reply]
Yeah, then I am afraid you need to ask the Sonos integration team.

**Comment by jdekoven** [comment]
I don't think it's Sonos because the que does show on Sonos app, just not the Sonos card.  I'll keep playing around with it though

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by punxaphil** [reply]
Neither do I. It's the Sonos integration: https://www.home-assistant.io/integrations/sonos/
